code-challenges.js

```diff
@@ -41,8 +83,27 @@ var vowelTester1 = "learn"
 var vowelTester2 = "throw"
 // Expected output: 3
 
-
-
+// create a function that accepts a string
+const findVowel = (string) => {
+    // make a list of vowels
+    let vowels = 'aeiou'
+    
+    // change string to all lowercase
+    string = string.toLowerCase()
+
+    // loop through string
+    for (let i = 0; i < string.length; i++) {
+        // find if the letter at each index of the string matches or has an index in the list of vowels
+        if (vowels.indexOf(string[i]) !== -1){
+            // if so, return the index and break out of the loop (so that it doesn't continue to try to find vowels)
+            return i
+            break
```

Comment:
I love the fact you're thinking about stoping the break, but because you used a return, that automatically stops the process and returns the index on the spot, therefore the break is never reached. If you try running the code without the break, it will work just fine! 

No need to refactor or change anything, just wanted to make you aware!